baybe/searchspace/continuous.py

```diff
@@ -180,12 +179,12 @@ def samples_random(self, n_points: int = 1) -> pd.DataFrame:
         """
         if not self.parameters:
             return pd.DataFrame()
-
+        import torch
```

Comment:
torch and botorch here are actually only necessary if there are constraints. If there are none and the lists are empty we can just sample without the `get_polytope_samples` which means torch and botorch wouldnt be needed

Can you make a TODO note so this is revisited?